pkg/dashboard/helpers_test.go

```diff
@@ -29,31 +30,43 @@ func Test_getUUID(t *testing.T) {
 }
 
 func Test_printResource(t *testing.T) {
+	// struct to contain test cases defined below
```

Comment:
Unnecessary comments

Reply:
It took me a long time after graduating to accept this, but [comments are considered harmful](https://blog.humphd.org/vocamus-894/)  😅